src/encoded/tests/test_clear_db_es_contents.py

```diff
@@ -22,56 +56,254 @@ def test_clear_db_tables(app, testapp):
+
+                    # test if we are only running on specific envs
+                    with logged_messages(error=[(f'clear_db_es_contents: The current environment, {test_env},'
+                                                 f' is not {disjoined_list(allowed_envs)}.'
+                                                 f' Skipping the attempt to clear DB.')]):
+                        assert run_clear_db_es(app, only_envs=allowed_envs, skip_es=True) is False
+                    expected_db_clears += 0
+                    expected_es_clears += 0
+                    assert mock_clear_db_tables.call_count == expected_db_clears
+                    assert mock_run_create_mapping.call_count == expected_es_clears
+
+                    # test again if we are only running on specific envs
+                    with logged_messages(error=[(f'clear_db_es_contents: The current environment, {test_env},'
+                                                 f' is not {disjoined_list(allowed_envs)}.'
+                                                 f' Skipping the attempt to clear DB.')]):
+                        assert run_clear_db_es(app, only_envs=allowed_envs, skip_es=False) is False
+                    expected_db_clears += 0
+                    expected_es_clears += 0
+                    assert mock_clear_db_tables.call_count == expected_db_clears
+                    assert mock_run_create_mapping.call_count == expected_es_clears
+
+                    # test if we are only running on specific envs
+                    assert run_clear_db_es(app, only_envs=['fourfront-test-env'], skip_es=True) is True
+                    expected_db_clears += 1
+                    expected_es_clears += 0
+                    assert mock_clear_db_tables.call_count == expected_db_clears
+                    assert mock_run_create_mapping.call_count == expected_es_clears
+
+                    # test again if we are only running on specific envs
+                    assert run_clear_db_es(app, only_envs=['fourfront-test-env'], skip_es=False) is True
+                    expected_db_clears += 1
+                    expected_es_clears += 1
+                    assert mock_clear_db_tables.call_count == expected_db_clears
+                    assert mock_run_create_mapping.call_count == expected_es_clears
+
+                    allowed_envs = ['fourfront-test-env-decoy-1', 'fourfront-test-env-decoy-2']
+                    # test if we are only running on specific envs
+                    with logged_messages(error=[(f'clear_db_es_contents: The current environment, {test_env},'
+                                                 f' is not {disjoined_list(allowed_envs)}.'
+                                                 f' Skipping the attempt to clear DB.')]):
+                        assert run_clear_db_es(app, only_envs=allowed_envs, skip_es=False) is False
+                    expected_db_clears += 0
+                    expected_es_clears += 0
```

Comment:
Same comment from CGAP - I get being explicit here but still feels unnecessary.

Reply:
If I didn't do this, I'd put the same thing commented out. It seems harmless, so I'm ignoring your uneasiness. :)

Reply:
The idea is so visually as you scan down the page, you can read off the number of expected changes for each section. The math is secondary to the self-documentation.